src/OpenSage.Game/Game.cs

```diff
@@ -691,16 +685,24 @@ private void StartGame(
         {
             for (var i = 0; i < playerSettings.Length; i++)
             {
-                Scene3D.CreateSkirmishPlayerStartingBuilding(
-                    playerSettings[i],
-                    Scene3D.Players[i + 2]);
+                var playerSetting = playerSettings[i];
+                var player = Scene3D.Players[i + 2];
+                var startingBuilding = Scene3D.CreateSkirmishPlayerStartingBuilding(playerSetting, player);
+
+                if (player.IsHuman)
+                {
+                    Scene3D.TacticalView.LookAt(startingBuilding.Transform.Translation);
+                    Scene3D.TacticalView.InitHeightForMap();
+                    Scene3D.TacticalView.SetAngleAndPitchToDefault();
+                    Scene3D.TacticalView.SetZoomToDefault();
+                }
             }
         }
 
         if (Scene3D.LocalPlayer.SelectedUnits.Count > 0)
         {
             var mainUnit = Scene3D.LocalPlayer.SelectedUnits.First();
-            Scene3D.CameraController.GoToObject(mainUnit);
+            // Scene3D.CameraController.GoToObject(mainUnit);
```

Comment:
Can this be removed? (I haven't finished reviewing yet so I don't know there's code further down that performs the same behavior.)